docs/testing/playwright/01-overview.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Playwright Overview
+sidebar_label: Overview
+---
+
+:::note
+The Stencil Playwright adapter is designed to only work with **version 4.13.0 and higher** of Stencil!
+:::
+
+[Playwright](https://playwright.dev/) is an automated end-to-end testing framework built to run on all modern browser engines and operating systems.
+Playwright leverages the DevTools protocol to provide reliable tests that run in actual browsers.
+
+## Set Up
+
+To add Playwright to an existing Stencil project, leverage the [Stencil Playwright testing adapter](https://www.npmjs.com/package/@stencil/playwright). This
+is a tool built by the Stencil team to help Stencil and Playwright work better together. The best part is you'll write your tests using the same APIs
+as defined and documented by Playwright. So, be sure to [check out their documentation](https://playwright.dev/docs/writing-tests) for help writing your first tests!
+
+To install the Stencil Playwright adapter in an existing Stencil project, follow these steps:
+
+1. Install the necessary dependencies:
+
+   ```bash npm2yarn
+   npm i @stencil/playwright @playwright/test --save-dev
+   ```
+
+1. Install the Playwright browser binaries:
+
+   ```bash
+   npx playwright install
+   ```
+
+1. Create a Playwright config at the root of your Stencil project:
+
+   ```ts title="playwright.config.ts"
+   import { expect } from '@playwright/test';
+   import { matchers, createStencilPlaywrightConfig } from '@stencil/playwright';
+
+   // Add custom Stencil matchers to Playwright assertions
+   expect.extend(matchers);
+
+   export default createStencilPlaywrightConfig();
```

Comment:
Maybe show case that folks can extend the PW config here:
```suggestion
   export default createStencilPlaywrightConfig({
       // overwrite Playwright config options here
       reporter: 'line',
       // ...
   });```

Reply:
Are there any configs that aren't allowed to be overwritten, people should be aware of?

Reply:
Nope, we allow all to be overwritten. That being said, there are some I'd advise against overwriting, so I'll add something around that.